packages/analytics-js-integrations/__tests__/integrations/GA4/browser.test.js

```diff
@@ -52,18 +52,23 @@ describe('Google Analytics 4 Track events tests', () => {
 });
 
 describe('Google Analytics 4 Page events tests', () => {
-  let ga4;
-  beforeEach(() => {
-    ga4 = new GA4(
-      { measurementId: 'G-123456', extendPageViewParams: true, capturePageView: 'rs', piiPropertiesToIgnore: [{ piiProperty: '' }] },
-      { getUserId: () => '1234', getUserTraits: () => { } },
-      destinationInfo,
-    );
-    ga4.init();
-    window.gtag = jest.fn();
-  });
+  pageEvents.forEach((event, index) => {
```

Comment:
You can use `it.each` for automatic enumeration of test cases.
[optional]

Reply:
will pick up later and do it for all ga4 tests